api/include/opentelemetry/trace/tracer.h

```diff
@@ -29,6 +30,7 @@ class Tracer
    * Attributes will be processed in order, previous attributes with the same
    * key will be overwritten.
    */
+
```

Comment:
I don't think the formatter will complain if you return this file to it's current state in master. If it does, there's something wrong with the formatter, and that's an issue that can be addressed later on. I don't think this file should be changed in this PR, I recommend changing these whitespaces back. The same applies to all other files in this PR that only have whitespace changes.

When I'm formatting my files, I do Ctrl-Shift-i in VSCode on all of the files I've modified. I've never run into formatting issues with this approach. The problem here might be that you're running the formatter on the entire project, rather than only on files you're modifying.

Reply:
I have removed the white space here.